### PR TITLE
QA fixes

### PR DESCRIPTION
**Pull request by driftingly**
This PR addresses issues discovered during QA.

## Fixes 

- Fixes issue with associating artwork with a prompt
- Fixes issue with 'Shape Face' not saving
- Fixes issue with `title` requirements and moves `title` input into the form fields section to improve the user experience when adding/editing `title`. 
- Makes artist name searchable

## Improvements

- Adds hidden field to prompts artwork for repeater title (thanks @nikhiltri for this great suggestion)
- Adds prompt navigation to sidebar of theme and individual prompts

Screenshot 2024-04-30 at 11 00 27 AM
Screenshot 2024-04-30 at 11 00 39 AM

### CODE REVIEW

**Review by nikhiltri — APPROVED** [review]
Looks good! ⭐

File: app/Http/Controllers/Twill/ThemeController.php [hunk 1]
```diff
@@ -36,11 +37,29 @@ public function getCreateForm(): Form
...
+            ->withFieldSets(new Fieldsets([
+                Fieldset::make()->title('Prompts')->id('prompts')->fields([
+                    BladePartial::make()
+                        ->view('forms.prompts')
+                        ->withAdditionalParams([
+                            'theme' => $model,
+                        ]),
+                ]),
+            ]));
+    }
```
Comment (zachgarwood) [inline app/Http/Controllers/Twill/ThemeController.php:52]: 👍

File: resources/views/forms/prompts.blade.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+<div class="custom">
+    <ul class="py-2">
+        @foreach($theme->prompts()->orderBy('position')->get() as $prompt)
+            <li class="text-lg">
+                <a
+                    @class(['block p-2  hover:bg-slate-100', 'font-semibold' => $prompt->id == $currentPromptId])
+                    href="{{ route('twill.themes.prompts.show', [$theme->id, $prompt->id]) }}"
+                >
+                    {{ ($prompt->id == $currentPromptId? '👉' : '') }}
```
Comment (zachgarwood) [inline resources/views/forms/prompts.blade.php:17]: 👉  👍

**Review by zachgarwood — APPROVED** [review]
Looks good

File: resources/views/forms/prompts.blade.php (hunk 2 above)
Comment (driftingly) [reply resources/views/forms/prompts.blade.php:17]: Typically I don't use emojis, but I've been using them a bit more. I'm finding them a quick way to call attention.

**Pull request merged**